packages/zcli-apps/src/lib/buildAppJSON.ts

```diff
@@ -84,6 +107,16 @@ export const getAppPayloadFromManifest = (appManifest: Manifest, port: number, a
   }
 }
 
+const getSettingsArr = (appSettings: any) => {
+  const s: any[] = []
+  Object.keys(appSettings).forEach((settingName: string) => {
+    const setting: Record<string, any> = {}
```

Comment:
I saw `settings` are coming from two places so I think it's safe to use `const setting: Record<string, string> = {}`
```
  const configSettings = manifest.parameters.reduce((result: Dictionary<string>, param) => {
    if (configContainsParam(param.name)) {
      result[param.name] = configParams[param.name] as string
    }
    return result
  }, {})
```

```
  const settings: Dictionary<string> = {}
  for (const param of params) {
    const value = await cli.prompt(`Enter ${appName} setting.${param.name} value`, { type: param.secure ? 'hide' : 'normal', required: valuesRequired })
    if (value) settings[param.name] = value
  }
  return settings
```